docs_user/modules/proc_deploying-backend-services.adoc

```diff
@@ -73,14 +65,14 @@ SWIFT_PASSWORD=$(cat ~/tripleo-standalone-passwords.yaml | grep ' SwiftPassword:
 
 .Procedure
 
-. Make sure you are using the {OpenShift} namespace where you want the
-control plane deployed:
+. Ensure that you are using the {OpenShift} namespace where you want the
+control plane to be deployed:
 +
 ----
-oc project openstack
+$ oc project openstack
 ----
 
-. Create OSP secret.
+. Create the {OpenStackShort} secret.
```

Comment:
Can we give an example of one common way to create the secret downstream?

Reply:
We should either link to section 2.2 "Providing secure access to the Red Hat OpenStack Services on OpenShift services" of the deployment guide or pull the same content in here: https://docs.redhat.com/en/documentation/red_hat_openstack_services_on_openshift/18.0-beta/html-single/deploying_red_hat_openstack_services_on_openshift/index#proc_providing-secure-access-to-the-RHOSO-services_preparing